lib/request.js

```diff
@@ -3,12 +3,12 @@
  * Module dependencies.
  */
 
-var contentType = require('content-type');
-var stringify = require('url').format;
-var parse = require('parseurl');
-var qs = require('querystring');
-var typeis = require('type-is');
-var fresh = require('fresh');
+const contentType = require('content-type');
+const stringify = require('url').format;
```

Comment:
@ruimarinho can't exactly tell what you're pointing to here. which line/statement/dependency?